client/src/legacy/CMSMain.EditForm.js

```diff
@@ -232,7 +232,7 @@ $.entwine('ss', function($){
 
       holder.next().filter('.listbox')[instant ? 'hide' : 'slideUp'](() => {
         holder.removeClass('field--merge-below');
-      });
+      }).css('overflow','visible'); // jquery adding overflow:hidden in hide, this will break listbox display
```

Comment:
awesome, could we add this to the line above as well?
`.slideUp()` has a delay which means this line ends up being overwritten